pandas/core/groupby/generic.py

```diff
@@ -733,13 +732,14 @@ def _selection_name(self):
        min  max
     1    1    2
     2    3    4
+    """)
 
+    _agg_see_also_doc = dedent("""
```

Comment:
small thing, but if you son't mind defining this before examples (that's the order in the docstring)

Reply:
No worries, have just done so across a couple of the changes with incorrect order, will push now.